hack/build/run-functional-tests.sh

```diff
@@ -16,11 +16,24 @@
 
 set -euo pipefail
 
+script_dir="$(readlink -f $(dirname $0))"
 source hack/build/config.sh
 source hack/build/common.sh
 
+# functional testing
+KUBECTL=${KUBECTL:-${CDI_DIR}/cluster/.kubectl}
+KUBECONFIG=${KUBECONFIG:-${CDI_DIR}/cluster/.kubeconfig}
+KUBE_MASTER_URL=${KUBE_MASTER_URL:-""}
+CDI_NAMESPACE=${CDI_NAMESPACE:-kubesystem}
+
```

Comment:
In pr 352 the env var is named KUBECTL_PATH and the readme shows that. Is this changing?
Also, what's the current status of an OC_PATH env var?
@copejon @awels

Reply:
For now i'm leaving OC_PATH in as it's still a flag in the tests.  It's inert, so there's not a need to remove it right now.  I'd prefer to do that in another PR.  I'll fix up the readme to reflect these changes.